cmds/task/fetchers.go

```diff
@@ -58,7 +58,45 @@ func runStatus(credentials *tcclient.Credentials, args []string, out io.Writer,
 	fmt.Fprintln(out, getRunStatusString(s.Status.Runs[runID].State, s.Status.Runs[runID].ReasonResolved))
 	return nil
 }
+// confirmMsg displays confirmation message when --confirm is used
+func confirmMsg(command string, credentials *tcclient.Credentials, args []string, out io.Writer, _ *pflag.FlagSet )(string) {
+	
+	q := makeQueue(credentials)
+	taskID := args[0]
+
+	c, _ := q.Status(taskID)
+	run := c.Status.Runs[len(c.Status.Runs)-1]
+
+	t, _ := q.Task(taskID)
+	
+	fmt.Println(command, t.Metadata.Name ,"taskid:", taskID ,"(state:",run.State,"). Are you sure you want to proceed with given command? (y/N)")
+	var response string
+	fmt.Scanf("%s", &response)
+
+
+	return response
+}
+
+// displayNoopMsg displays details when --noop is used
+func displayNoopMsg(command string, credentials *tcclient.Credentials, args []string, out io.Writer, _ *pflag.FlagSet ) error {
```

Comment:
like `confirmMsg`, there's no need to pass unused parameters here (`out` and the FlagSet), and it would be better to pass the taskId directly, rather than args.

Reply:
Also, `command` here seems to be a capitalized continuing-present-tense verb like "Completes".  Since this is a no-op, perhaps something like `"Would complete %s - taskId %s, state %s"` would be a bit better.  This variable might be better named `action` or `verb` in this case.